patches/tModLoader/Terraria/GameContent/UI/States/UIWorkshopHub.TML.cs

```diff
@@ -50,16 +50,5 @@ partial class UIWorkshopHub
 			Interface.modBrowser.PreviousUIState = this;
 			Main.MenuUI.SetState(Interface.modBrowser);
 		}
-
-		private UIElement MakeButton_TBD() {
```

Comment:
Please comment out this code instead of removing it - we may reuse this later if we find an agreed on use case for the fourth button.